components/org.wso2.carbon.identity.local.auth.api.endpoint/src/main/resources/api.identity.local.auth.yaml

```diff
@@ -58,6 +58,11 @@ paths:
       description: |
         This API is used to authenticate the user and to get a JWT that can be used to identify the user authenticated
       x-wso2-request: |
+        curl -k -v -X POST -H "Authorization: Basic <Base64Encoded[username:password]>" -H "Content-Type: application/json"
+        "https://localhost:9443/api/identity/auth/v1.2/authenticate"
+        curl -k -v -X POST -H "Content-Type: application/json" -d '{ "[username]": "[admin]","[password]": "[admin]"}'
```

Comment:
Shall we change this to
`-d '{ "[username]": "[username]","[password]": "[password]"}'` as we are showing two placeholders

Reply:
corrected it now.